inlong-sort/sort-common/src/test/java/org/apache/inlong/sort/protocol/transformation/relation/NodeRelationTest.java

```diff
@@ -25,10 +25,15 @@
 import static org.junit.Assert.assertEquals;
 
 /**
- * Tests for{@link NodeRelationTest}
+ * Tests for {@link NodeRelationTest}
```

Comment:
`NodeRelationTest` should be NodeRelation

Reply:
done